src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueDelegate.scala

```diff
@@ -28,59 +28,33 @@ private[launchqueue] class LaunchQueueDelegate(
 
   val launchQueueRequestTimeout: Timeout = config.launchQueueRequestTimeout().milliseconds
 
-  override def list: Seq[QueuedInstanceInfo] = {
-    askQueueActor[LaunchQueueDelegate.Request, Seq[QueuedInstanceInfo]]("list")(LaunchQueueDelegate.List)
-  }
-
   override def listAsync: Future[Seq[QueuedInstanceInfo]] =
     askQueueActorFuture[LaunchQueueDelegate.Request, Seq[QueuedInstanceInfo]]("list")(LaunchQueueDelegate.List)
 
-  override def listWithStatistics: Seq[QueuedInstanceInfoWithStatistics] = {
-    askQueueActor[LaunchQueueDelegate.Request, Seq[QueuedInstanceInfoWithStatistics]]("listWithStatistics")(LaunchQueueDelegate.ListWithStatistics)
-  }
-
   override def listWithStatisticsAsync: Future[Seq[QueuedInstanceInfoWithStatistics]] =
     askQueueActorFuture[LaunchQueueDelegate.Request, Seq[QueuedInstanceInfoWithStatistics]]("listWithStatistics")(LaunchQueueDelegate.ListWithStatistics)
 
-  override def get(runSpecId: PathId): Option[QueuedInstanceInfo] =
-    askQueueActor[LaunchQueueDelegate.Request, Option[QueuedInstanceInfo]]("get")(LaunchQueueDelegate.Count(runSpecId))
-
   override def getAsync(runSpecId: PathId): Future[Option[QueuedInstanceInfo]] =
```

Comment:
can we now remove the word `Async` from all method names since this is the way we do stuff in all methods? :)

Reply:
I'd do this in a second step.